tools/topology/topology2/cavs-mixin-mixout-eqiir-dts-ssp.conf

```diff
@@ -0,0 +1,220 @@
+#
+# List of all DAIs
+#
+#SSP Index: 0, Direction: duplex
+#Object.Dai.SSP [
```

Comment:
duplicated in mix.conf. Remove SSP part.

Reply:
@macchian The duplicated parts are removed now.